src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -401,10 +403,17 @@ public QueryResult query(QueryRequest queryRequest) {
     }
 
     /**
-     * TODO: write Javadoc.
+     * Execute a scan meeting the specs of @{link AmazonDynamoDB}, but scoped to single tenants within a shared table.
+     * If no tenant context is specified, a multi tenant scan is performed over the multi tenant shared table.
+     * Scans scoped to single tenants is not a performant operation.
+     * Multi tenant scans are as bad as scans on any other dynamo table, ie: not great.
```

Comment:
"ie:" -> "i.e.,"

Reply:
"ie:" is shorter and gets the same message across. Agree to disagree?

Reply:
the Salesforce Style Guide (as well basically every other style guide (including the broadly accepted Chicago Manual of Style)) insists on "i.e." as the spelling for the abbreviation of "id est" (and also insists that "i.e." should be followed by a comma). (FWIW, the Salesforce Style Guide goes one further and at least suggests avoiding "i.e." entirely—e.g., replacing it with something like "that is" or "in other words".)

while the actual code is where the most important work is, documentation—especially public-facing (open-source) documentation—adhering to a uniform style enhances clarity and readability.